db/migrate/20240422145748_add_page_to_page_views.rb

```diff
@@ -0,0 +1,7 @@
+class AddPageToPageViews < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def change
+    add_reference :page_views, :page, null: true, index: {algorithm: :concurrently}
+  end
+end
```

Comment:
I'm a little bit conflicted on this. The ideal way to go about this is to make a polymorphic association/model that will then allow any model to be in relation with PageView but I can see this to be a quick way to get this feature implemented first.

Reply:
Yup, I discussed this with @benhalpern. Data migration was the main concern with a polymorphic approach.

Are there any ways of sidestepping this issue if we went down a polymorphic approach?

Reply:
Yeah I think this has to be the first take on this as the path forward